hca/upload/cli/generate_status_report_command.py

```diff
@@ -0,0 +1,29 @@
+from hca.upload.upload_submission_state import UploadAreaFilesStatusCheck
+
+from hca.upload.cli.common import UploadCLICommand
+
+
+class GenerateStatusReportCommand(UploadCLICommand):
+    """
+    Generate file status report for upload area
+    """
+    @classmethod
+    def add_parser(cls, upload_subparsers):
+        gen_file_status_report_parser = upload_subparsers.add_parser('report', help=cls.__doc__,
+                                                                     description=cls.__doc__)
+        gen_file_status_report_parser.set_defaults(entry_point=GenerateStatusReportCommand)
+        gen_file_status_report_parser.add_argument('env', help="Environment the upload area was created in")
+        gen_file_status_report_parser.add_argument('uuid', help="Full UUID of an upload area.")
```

Comment:
Could not perhaps the env and uuid be optional arguments, and default to the env configured in .config/hca/config.json and the current upload area, respectively?

I imagine that the usual usage of this file will be to check the status of files you just uploaded. In that case having to specify env and area is overkill.

Reply:
👍